src/babashka/bbin/scripts.clj

```diff
@@ -281,19 +281,18 @@ exec bb \\
 (let [SCRIPT_LIB     '{{script/lib}}
       SCRIPT_COORDS  {{script/coords|str}}
       SCRIPT_MAIN_OPTS_FIRST  {{script/main-opts.0|pr-str}}
-      SCRIPT_MAIN_OPTS_SECOND {{script/main-opts.1|pr-str}} 
+      SCRIPT_MAIN_OPTS_SECOND {{script/main-opts.1|pr-str}}
       TMP_EDN (doto (fs/file (fs/temp-dir) (str (gensym \"bbin\")))
                  (spit (str \"{:deps {\" SCRIPT_LIB SCRIPT_COORDS \"}}\"))
                  (fs/delete-on-exit))]
-                                         
-(process/exec
-  (str/join \" \" (concat [\"bb --config\" (str TMP_EDN)
-                   SCRIPT_MAIN_OPTS_FIRST SCRIPT_MAIN_OPTS_SECOND
-                   \"--\"] *command-line-args*))))\""
-;
-; non-windows script template
-;  
-  "#!/usr/bin/env bash
+(process/exex
```

Comment:
Small typo here, though this should have been caught by the tests. I'm going to remove the Bash scripts now and just use the Babashka-based versions so we only have one set of script templates to maintain. Otherwise we'll have to have tests for both versions.

Reply:
Sounds like a plan